### ISSUE TITLE
EVE Valkyrie not launching

### ISSUE THREAD

**Issue by WolfEdmunds** [LibreVR/Revive#197]
I've tried launching from the Revive tab in the headset, I've tried from the desktop, I've tried using the injector, I've even tried the shortcut with modified target string to create an account and what not. Nothing works. 

Occasionally I'll get a flash on my pc screen, but not the headset, with an error of, "Eve Valkyrie must be run via the launcher. The game will now open the launcher for you", at which point nothing happens. I've gone through everything, would you mind pointing me in the right steps to get you fail files so you could potentially help me get this guy running?

**Comment by LibreVR** [comment]
Please send me the logs from `%APPDATA%\Revive`.

**Comment by WolfEdmunds** [comment]
From Revive Overlay:

libpng warning: iCCP: known incorrect sRGB profile
Loaded manifest from: C:\Program Files\Revive\revive.vrmanifest
Runtime directory: file:///D:/Steam/steamapps/common/SteamVR/
Oculus directory: file:///C:/Program Files/Oculus/
Loading assets manifest: file:///c:/Program Files/Oculus/Software/Manifests/ccp-games-newcastle-eve-valkyrie_assets.json
Loading assets manifest: file:///c:/Program Files/Oculus/Software/Manifests/playful-luckys-tale_assets.json
Found assets bundle ccp-games-newcastle-eve-valkyrie_assets
Loading application manifest: file:///C:/Program Files/Oculus/Software/Manifests/ccp-games-newcastle-eve-valkyrie.json
Found assets bundle playful-luckys-tale_assets
Loading application manifest: file:///C:/Program Files/Oculus/Software/Manifests/playful-luckys-tale.json
Found application ccp-games-newcastle-eve-valkyrie
Found application playful-luckys-tale
Launching application: ccp-games-newcastle-eve-valkyrie
Launching application: ccp-games-newcastle-eve-valkyrie
Failed to launch application: revive.app.ccp-games-newcastle-eve-valkyrie ()


From Revive Injector:

Launched injector with: "C:\Program Files\Oculus\Software\Software\ccp-games-newcastle-eve-valkyrie\Image\WindowsNoEditor\VkGame\Binaries\Win64\ReviveInjector_x64.exe" "C:\Program Files\Oculus\Software\Software\ccp-games-newcastle-eve-valkyrie\Image\WindowsNoEditor\VkGame\Binaries\Win64\EVE Valkyrie.exe"
Creating process: C:\Program Files\Oculus\Software\Software\ccp-games-newcastle-eve-valkyrie\Image\WindowsNoEditor\VkGame\Binaries\Win64\EVE Valkyrie.exe 
Injecting DLL: C:\Program Files\Oculus\Software\Software\ccp-games-newcastle-eve-valkyrie\Image\WindowsNoEditor\VkGame\Binaries\Win64\x64\openvr_api.dll
LoadLibrary found at address: 0x90eb74d0
Injecting DLL: C:\Program Files\Oculus\Software\Software\ccp-games-newcastle-eve-valkyrie\Image\WindowsNoEditor\VkGame\Binaries\Win64\x64\LibRevive64_1.dll
LoadLibrary found at address: 0x90eb74d0
Injected dlls succesfully

**Comment by LibreVR** [comment]
You must use the Revive Dashboard to launch EVE Valkyrie. Also, try reinstalling Revive to make sure you are running the latest version.

**Comment by WolfEdmunds** [comment]
For clarification, the Revive dashboard is the pin on the bottom next to desktop and steam, correct? Also, I suppose it's possible it's an old version, I'm pretty sure I double checked though. I'll report back with any developments.  Thanks for your help thus far.

**Comment by djsmrk** [comment]
Im going through the same thing . I also tried dropping the .exe for eve into the injector. still comes up with the same "Run via the launcher " problem  . This was not a free game . figures all the free games worked . this was the first one I bought .

**Comment by LibreVR** [comment]
@djsmrk First, make sure you are running the latest version of Revive, reinstall it to be sure. Then try launching it from the dashboard.

If you are using the latest version and are still running into this problem, then go through the following:

1. Extract the injector in a folder, anywhere is fine.
2. Create a batch file in that folder with the following contents: `ReviveInjector_x64.exe /base Software\Software\ccp-games-newcastle-eve-valkyrie\Image\WindowsNoEditor\VkGame\Binaries\Win64\EVE Valkyrie.exe -oculus`
3. Execute the batch file.

This is a temporary solution, the Revive Dashboard should be able to take care of this automatically.

**Comment by LibreVR** [comment]
@WolfEdmunds That is correct, the Revive dashboard is the button next to Steam and Desktop.

Please give me a Revive Injector log file after you tried starting it from that dashboard. Make sure you're running the latest version.

**Comment by djsmrk** [comment]
thank you libre . Ill try it as soon as I get home. wish me luck

**Comment by WolfEdmunds** [comment]
Here's the Injector file info. I couldn't find the version number, but re-installed Revive, and then attempted to run Eve Valkyrie from the Vive, and nothing happened. 

Launched injector with: C:\Program Files\Revive\Revive\ReviveInjector_x64.exe /base "Software/Software/ccp-games-newcastle-eve-valkyrie/Image/WindowsNoEditor/VkGame/Binaries/Win64/EVE Valkyrie.exe" -oculus -vr -tenant=live
Creating process: C:\Program Files\Oculus\Software\Software\ccp-games-newcastle-eve-valkyrie\Image\WindowsNoEditor\VkGame\Binaries\Win64\EVE Valkyrie.exe -oculus -vr -tenant=live 
Injecting DLL: C:\Program Files\Revive\Revive\x64\openvr_api.dll
LoadLibrary found at address: 0x517574d0
Injecting DLL: C:\Program Files\Revive\Revive\x64\LibRevive64_1.dll
LoadLibrary found at address: 0x517574d0
Injected dlls succesfully

**Comment by LibreVR** [comment]
That is the correct log. I'll double check whether EVE is still working correctly tomorrow.

Also, I assume you are using an HTC Vive and not an other headset?

**Comment by WolfEdmunds** [comment]
Yes, using the vive. Is there a way to check the version I'm using of Revive? Hovering over the .exe just shows 0.0.0.0. I'm just downloading form the link on the main page.

**Comment by LibreVR** [comment]
Version information is not added yet, the link from the main page is the latest version. Version information will be added once we hit version 1.0.

**Comment by WolfEdmunds** [comment]
Also, I wrote the .bat file and ran it, I'm stuck on "Connecting to the Cloud."

**Comment by noodle1009** [comment]
FWIW, I'm able to run this without issue in 0.8.3, launched from the Revive's overlay within SteamVR.

It took a while to load up, but it always has for me at least.  Just played through the tutorial mission to make sure.

**Comment by WolfEdmunds** [comment]
How long is a while?

**Comment by LibreVR** [comment]
@WolfEdmunds I think you may be blocking the Oculus servers or just have trouble connecting to them.

**Comment by WolfEdmunds** [comment]
No AV installed, and Windows defender is turned off.

**Comment by WolfEdmunds** [comment]
Why does the Revive Overlay file always state failed at the end? Is that relevant?

**Comment by WolfEdmunds** [comment]
<img width="2048" alt="untitled" src="https://cloud.githubusercontent.com/assets/20359871/16710060/455a2b58-45f0-11e6-8884-b412b99c4ecc.png">

**Comment by WolfEdmunds** [comment]
This is what it shows on the PC, so it is probably stuck where you said it is, trying to connect. I'm going to DMZ my router for a min and see if that fixes it.

**Comment by WolfEdmunds** [comment]
DMZing my router did nothing. No idea what to do.

**Comment by noodle1009** [comment]
it took nearly 60 seconds for it to start for me, using the revive overlay within SteamVR.  I have it installed on a mechanical disk drive, not a SSD, and I have an i7-4790K with a 980Ti.  Not sure if it takes longer than that with other equipment, but it's definitely slower than what I'm used to in launching.  

I've already created / linked an EVE Valyrie account (required to play the game) so maybe the problem is with the step prior?  If memory serves, if it's the first time you've played the game, you reach a screen where you're instructed to take off the HMD and go to a website to register (https://account.evevalkyrie.com/signup).  Maybe for my setup it's skipped and that is where things are getting hung up.

**Comment by WolfEdmunds** [comment]
Mine is logged in, and when using the .bat file indicated above, I can get the game to show on the desktop, but just sit at connecting to the cloud.

**Comment by WolfEdmunds** [comment]
If I start through the Headset and through the Revive tab, it sits in the gray grid room, and on my pc, that is what I see. Unless I try to click EVE again, in which case it stops trying to start and reverts back to ready. Then, in order to get it to attempt to start again from the in-headset tab, I have to restart all three programs, and try again, else it will just flash a black box on the screen and thats it.

**Comment by djsmrk** [comment]
I'm sorry but I'm not sure how to create a batch file .

Thank You,
David DePietto
631-831-4627

**Comment by RandomMaster** [comment]
Hi Dj Smrk

1) Basically browse to the place where your ReviveInjector_X64.exe file is (it's wherever you installed Revive to in the first place I believe)
2) Right click in a blank space in that folder, go to new and create a new text document (txt) file, open it to edit it. 
3) Copy paste the line into that document: ReviveInjector_x64.exe /base Software\Software\ccp-games-newcastle-eve-valkyrie\Image\WindowsNoEditor\VkGame\Binaries\Win64\EVE Valkyrie.exe -oculus
4) Save the file.
5) Edit the text file to be whatever.bat (instead of New Text Document.txt)

Merely run that as you would any other file to complete the step mentioned by LIbreVR

**Comment by WolfEdmunds** [comment]
OK, Valkyrie is working. When you said it took a minute to launch, you weren't kidding. After disabling Windows Defender multiple times (damn thing loves to turn itself back on), I finally booted up the ocukus home, then the steam vr for the vive, then revive. Clicked on EVE once. No reaction.  Second click and the dashboard closed. When I reopened it after a few seconds it said now playing EVE Valkyrie. So I decided to take a shower. My wife kept checking it for me, and it ended up taking a out 5 mind at first launch. Now it only takes about 10-20 seconds. Thank you all for your help, and especially to you LibreVR. Hell of a job you did here!
